agent/api/container/container.go

```diff
@@ -26,6 +26,7 @@ import (
 	apierrors "github.com/aws/amazon-ecs-agent/agent/api/errors"
 	resourcestatus "github.com/aws/amazon-ecs-agent/agent/taskresource/status"
 	"github.com/aws/amazon-ecs-agent/ecs-agent/credentials"
+	"github.com/aws/amazon-ecs-agent/ecs-agent/logger"
```

Comment:
Q. We have some log field defined in [logger/field/constants.go](https://github.com/aws/amazon-ecs-agent/blob/dev/ecs-agent/logger/field/constants.go), should be add new useful fields there and reuse them? Thanks!